### DISCUSSION TITLE
[REFLET] Reflet loi d'eau

### DISCUSSION THREAD

**Discussion by ABIAdrienG** [witsa/synapps#667] [General]
Bonjour,
Depuis le passage en 1.2.0 nous n'arrivons plus à afficher correctement le reflet loi d'eau.
![image](https://github.com/witsa/synapps/assets/127521459/03ca10d7-15cb-4c38-8b9f-24ec78c99f46)
La représentation de la courbe semble ne plus être liée à la taille de police. 
Quel est le nouveau facteur influençant la taille du graphique ?

Je vous remercie par avance !

A.G.

**Comment by manufitoussiwit** [comment]
La suite dans l'issue #669
